nixos/modules/services/infrastructure/terraform-cloud-agent/default.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.terraform-cloud-agent;
+in
+{
+  options.services.terraform-cloud-agent = {
```

Comment:
The only other suggestion I have is to add more examples and descriptions to these options for upstreaming purposes, but that can be done later if this is urgent.

Reply:
Addressed in 14bc0131540c.